include/release-cycle.json

```
@@ -1,8 +1,16 @@
 {
-  "3.12": {
+  "3.13": {
     "branch": "main",
     "pep": 693,
     "status": "feature",
+    "first_release": "2024-10-07",
+    "end_of_life": "2029-10",
+    "release_manager": "Thomas Wouters"
+  },
+  "3.12": {
+    "branch": "3.12",
+    "pep": 693,
+    "status": "bugfix",
```

See https://github.com/python/devguide/blob/main/versions.rst?plain=1#L45

```suggestion
    "status": "prerelease",
```